test/transforms_test.dart

```diff
@@ -15,6 +15,13 @@ void main() {
     expect(res.data[3]['username'], 'awailas');
   });
 
+  test('order on multiple columns', () async {
```

Comment:
Same with this one, there's nothing wrong with this test, but just curious to why you changed the test from the one in supabase-js.